### ISSUE TITLE
Table : When selected, entire row information is not conveyed by screen reader.

### ISSUE THREAD

**Issue by DikshaHub** [SAP/ui5-webcomponents-react#5480]
### Describe the bug

When we use a Voice Over software, such as JAWS on Windows and navigate to a row in the table, entire row information is not conveyed . Also it doesn't convey if any field inside the row is blank . This issue was raised by accessibility testing team. 

### Isolated Example

https://codesandbox.io/p/sandbox/charming-lena-78pr5k?file=%2Fsrc%2FApp.js%3A20%2C9

### Reproduction steps

1. Open the link.
2. Open JAWS software.
3. Select the first row and check the speech history.
4. Select the second row and check the speech history.
5. After selecting the second row, press TAB and check the speech history again.

### Expected Behaviour

1. When the first row is selected, it should reflect "From" and "To" in the speech history or audio. But it only reads "From" ("To" is skipped).
2. After selecting second row ,  entire row information should be conveyed by screen reader. But it doesn't read anything. 
3. Though when you press TAB and enter tablecell , it reads out that particular cell is blank. 

### Screenshots or Videos

<img width="714" alt="MicrosoftTeams-image (1)" src="https://github.com/SAP/ui5-webcomponents-react/assets/66899235/1cbd6552-6b58-4870-a4d3-12359658b75f">

### UI5 Web Components for React Version

~1.19.0

### UI5 Web Components Version

~1.17.0

### Browser

Chrome

### Operating System

Windows

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.
